packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -67,6 +67,7 @@ export interface IOpsCachingPolicy {
  */
 export interface ICollabSessionOptions {
 	/**
+	 * @deprecated starting in 2.0-RC3. No longer needed.
 	 * Value indicating the display name for session that admits unauthenticated user.
 	 * This name will be used in attribution associated with edits made by such user.
```

Comment:
The deprecated comment should appear last (after the description), otherwise all of the documentation here is treated as part of the deprecation instructions, and not as the description of the item.
```suggestion
	 * Value indicating the display name for session that admits unauthenticated user.
	 * This name will be used in attribution associated with edits made by such user.
	 * @deprecated starting in 2.0-RC3. No longer needed.
```

Reply:
This is why the item now appears as "(undocumented)" in the API report

Reply:
@pragya91 @Josmithr I think this got auto-merged without addressing these documentation regressions that Josh identified in his PR feedback.  Is there any PR and/or ADO item tracking repairing these?

Reply:
@ChumpChief @aboktor had created a followup item here: https://office.visualstudio.com/OC/_workitems/edit/9126002
I had marked the pull request to auto-complete because of which we missed addressing of the comments.

Reply:
Thanks, will move discussion to that work item.